server/src/main/java/io/crate/expression/scalar/arithmetic/AbsFunction.java

```diff
@@ -53,5 +57,21 @@ public static void register(Functions.Builder builder) {
                 }
             );
         }
+        builder.add(
+            Signature.builder(NAME, FunctionType.SCALAR)
+                .argumentTypes(DataTypes.NUMERIC.getTypeSignature())
+                .returnType(DataTypes.NUMERIC.getTypeSignature())
+                .features(Scalar.Feature.DETERMINISTIC, Scalar.Feature.STRICTNULL)
+                .build(),
+            (signature, boundSignature) -> {
+                DataType<?> argType = boundSignature.argTypes().get(0);
```

Comment:
I guess getting boundSignature argType and using it as a return type is required to preserve arg params?
Can we add a short code comment here and in there places ? Similar to how it's done in `NumericSumAggregation`.

Reply:
yep, thx!